apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v1/eobReadCarrierMultipleLines.json

```diff
@@ -0,0 +1,2139 @@
+        "value" : 37.5,
+        "system" : "urn:iso:std:iso:4217",
+        "code" : "USD"
+      }
+    }, {
+      "category" : {
+        "coding" : [ {
+          "system" : "https://bluebutton.cms.gov/resources/codesystem/adjudication",
+          "code" : "https://bluebutton.cms.gov/resources/variables/line_bene_ptb_ddctbl_amt",
+          "display" : "Line Beneficiary Part B Deductible Amount"
+        } ]
+      },
+      "amount" : {
+        "value" : 0.0,
+        "system" : "urn:iso:std:iso:4217",
+        "code" : "USD"
+      }
+    }, {
+      "category" : {
+        "coding" : [ {
+          "system" : "https://bluebutton.cms.gov/resources/codesystem/adjudication",
+          "code" : "https://bluebutton.cms.gov/resources/variables/line_bene_prmry_pyr_pd_amt",
+          "display" : "Line Primary Payer (if not Medicare) Paid Amount"
+        } ]
+      },
+      "amount" : {
+        "value" : 0.0,
+        "system" : "urn:iso:std:iso:4217",
+        "code" : "USD"
+      }
+    }, {
+      "category" : {
+        "coding" : [ {
+          "system" : "https://bluebutton.cms.gov/resources/codesystem/adjudication",
+          "code" : "https://bluebutton.cms.gov/resources/variables/line_coinsrnc_amt",
+          "display" : "Line Beneficiary Coinsurance Amount"
+        } ]
+      },
+      "amount" : {
+        "value" : 9.57,
+        "system" : "urn:iso:std:iso:4217",
+        "code" : "USD"
+      }
+    }, {
+      "category" : {
+        "coding" : [ {
+          "system" : "https://bluebutton.cms.gov/resources/codesystem/adjudication",
+          "code" : "https://bluebutton.cms.gov/resources/variables/line_sbmtd_chrg_amt",
+          "display" : "Line Submitted Charge Amount"
+        } ]
+      },
+      "amount" : {
+        "value" : 35.0,
+        "system" : "urn:iso:std:iso:4217",
+        "code" : "USD"
+      }
+    }, {
+      "category" : {
+        "coding" : [ {
+          "system" : "https://bluebutton.cms.gov/resources/codesystem/adjudication",
+          "code" : "https://bluebutton.cms.gov/resources/variables/line_alowd_chrg_amt",
+          "display" : "Line Allowed Charge Amount"
+        } ]
+      },
+      "amount" : {
+        "value" : 47.84,
+        "system" : "urn:iso:std:iso:4217",
+        "code" : "USD"
+      }
+    }, {
+      "category" : {
+        "coding" : [ {
+          "system" : "https://bluebutton.cms.gov/resources/codesystem/adjudication",
+          "code" : "https://bluebutton.cms.gov/resources/variables/line_prcsg_ind_cd",
+          "display" : "Line Processing Indicator Code"
+        } ]
+      },
+      "reason" : {
+        "coding" : [ {
+          "system" : "https://bluebutton.cms.gov/resources/variables/line_prcsg_ind_cd",
+          "code" : "A",
+          "display" : "Allowed"
+        } ]
+      }
+    } ]
+  } ],
+  "payment" : {
+    "amount" : {
+      "value" : 199.99,
+      "system" : "urn:iso:std:iso:4217",
+      "code" : "USD"
+    }
+  }
+}
```

Comment:
Just want to confirm with @williamharty that we do want to make this change on v1 as well as v2 since we don't always do that.

Reply:
Laila Sharshar reported the error in V2.  I was not clear if it was happening in V1 as well. However, since this is a data integrity bug, I would support fixing it in V1 if needed.

Reply:
thanks, I'm going to confirm with PO as well

Reply:
confirmed